grails/start-grails

```diff
@@ -5,9 +5,8 @@
 # license that can be found in the LICENSE file.
 
 XMX=${JAVA_MAX_MEMORY-256}
-MAXPERM=${JAVA_MAX_PERM-128}
 JAVA_AGENT=${JAVA_AGENT}
 
 cd /home/application/current/build/libs
-java -Dgrails.env=prod -jar current-${GRAILS_APP_VERSION}.war -XX:MaxPermSize=${MAXPERM}m -Xmx${XMX}m ${JAVA_AGENT} --server.host=0.0.0.0 --server.port=${PORT}
+java -Dgrails.env=prod -jar -Xmx${XMX}m ${JAVA_AGENT} current-${GRAILS_APP_VERSION}.war --server.host=0.0.0.0 --server.port=${PORT}
```

Comment:
Did you test this?
According to `java -help` it should be:
java -Dgrails.env=prod -Xmx${XMX}m ${JAVA_AGENT} -jar current-${GRAILS_APP_VERSION}.war --server.host=0.0.0.0 --server.port=${PORT}
After the option -jar the jar file should be specified.
Correct me if I'm wrong.